src/Symfony/Component/DependencyInjection/Tests/ParameterBag/ParameterBagTest.php

```diff
@@ -84,7 +84,12 @@ public function testGetThrowParameterNotFoundException($parameterKey, $exception
             'fiz' => array('bar' => array('boo' => 12)),
         ));
 
-        $this->setExpectedException('Symfony\Component\DependencyInjection\Exception\ParameterNotFoundException', $exceptionMessage);
+        if (method_exists($this, 'expectException')) {
+            $this->expectException('Symfony\Component\DependencyInjection\Exception\ParameterNotFoundException');
```

Comment:
but only those changed in this PR, not the other one that should be kept asis to reduce future merge conflicts